extractor/src/main/java/org/schabi/newpipe/extractor/services/bandcamp/linkHandler/BandcampPlaylistLinkHandlerFactory.java

```diff
@@ -0,0 +1,31 @@
+// Created by Fynn Godau 2019, licensed GNU GPL version 3 or later
+
+package org.schabi.newpipe.extractor.services.bandcamp.linkHandler;
+
+import org.schabi.newpipe.extractor.exceptions.ParsingException;
+import org.schabi.newpipe.extractor.linkhandler.ListLinkHandlerFactory;
+
+import java.util.List;
+
+/**
+ * Just as with streams, the album ids are essentially useless for us.
+ */
```

Comment:
I think you should still just return the actual ID in `getId()`

Reply:
This would require querying the page, which I think would be better to avoid because the ID is not necessary.